backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/utils/MapCsvToStore.kt

```diff
@@ -0,0 +1,17 @@
+package app.ehrenamtskarte.backend.stores.utils
+
+import app.ehrenamtskarte.backend.stores.COUNTRY_CODE
+import app.ehrenamtskarte.backend.stores.importer.common.types.AcceptingStore
+import app.ehrenamtskarte.backend.stores.webservice.schema.types.CSVAcceptingStore
+
+fun getDiscount(discounts: MutableList<String?>): String {
+    discounts.removeIf { it.isNullOrEmpty() }
+    return discounts.joinToString("\n\n")
+}
+fun mapCsvToStore(csvStore: CSVAcceptingStore): AcceptingStore {
```

Comment:
optional but nice to have I think:
do we want to handle an error, when csvStore failed to be mapped into AcceptingStore?
it can happen for example if categoryId can't be parsed into Integer, or coordinates cannot be parsed into Double.
currently the service returns 500 internal server error in such case.

Reply:
you can add an improvement github issue. I think this pr is now big enough :)